src/components/shared/frame/UserList/index.tsx

```diff
@@ -87,7 +88,7 @@ export const UserList = ({
   const searchInputData = useSelector(appManagementSelector)
 
   const validateSearchText = (expr: string) => {
-    const validateExpr = /^[ A-Za-z0-9._!@-]*$/.test(expr)
+    const validateExpr = Patterns.EMAIL_SEARCH.test(expr)
```

Comment:
I guess this should better use the function `isSearchUserEmail(expr)` from Patterns.ts below

Reply:
Hi @oyo, Can you please review it? 
I made a changes according to your suggestions.